templates/web/pages/product/includes/product_image.tt.html

```diff
@@ -7,6 +7,9 @@
 		<p>[% creativecommons %][% original_link %]</p>
 		<p>[% attribution %]</p>
 	</figcaption>
+	<a class="button small action-report_product_to_nutripatrol" href="https://nutripatrol.openfoodfacts.org/flag/image/?barcode=[% code %]&image_id=[% original_id %]&source=web&flavor=off">
```

Comment:
"off" should not be hardcoded, you can use [% flavor %]

The string "Signaler ce produit à nos modérateurs" should be "Signal this image to our moderators" or something similar, and added to po/common.pot and po/en.po